harness/determined/pytorch/_pytorch_context.py

```diff
@@ -102,6 +114,8 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
             bool, optimizations_config.get("average_training_metrics")
         )
 
+        self.tbd_writer = None  # type: Optional[tensorboard.MetricWriter]
```

Comment:
since `PyTorchContext` is a public-facing object, you must not add private variables without an underscore prefix.

There are legacy properties that should have an underscore prefix, but don't... which is unfortunate, but new properties can do better than that.